en/includes/guides/applications/register-technology-wise-app.md

```diff
@@ -0,0 +1,40 @@
+# Register a {{ technology }} Application
+
+{{ product_name }} provides an application template with default configurations to register a {{ technology }} application with minimal effort.
+
+To register a {{ technology }} application,  
+
+1. On the {{ product_name }} Console, go to **Applications**.
+
+2. Click **New Application** and select **{{ technology }}**. The application creation wizard will appear.
+
+    ![Select {{ technology }} application template]({{base_path}}/assets/img/guides/applications/select-technology-app-template.png){: width="700" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+3. Enter the following details:
```

Comment:
```suggestion
3. In the wizard that appears, enter the following details:
```

Reply:
Resolved via https://github.com/wso2/docs-is/pull/5412/commits/601fe63e420f74d58339e442381c9b38541e840e